### PR TITLE
22717-adjust wordings based on the feedback doc and discussion

### PR DESCRIPTION
**Pull request by eason-pan-bc**
*Issue #:* /bcgov/entity#22717

*Description of changes:*
- Adjusted the business overall description
Frame 1
- Adjusted the description for multiple endpoints
Frame 2

_Note:_
Regarding the "No Methods" comment, as per discussion with Dylan, we will leave it as what it is right now. Since all the schemas used for filing are used in the Filing schema, and the the Filing schema is the one directly used in related endpoints. Also cannot remove "No methods use this type." since it's an Apigee setting.

Test Apigee instance: https://storied-landing-429920-d2-apispecs.apigee.io/docs/apispecstest/1/overview

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: docs/business.yaml [hunk 1]
```diff
@@ -3,8 +3,33 @@ info:
...
+    - AGM Extension Information 
+    - AGM Location Change Information 
+    - Alteration 
+    - Annual Report 
+    - Change of Address
+    - Change of Directors
+    - Change of Entity Name
+    - Change of Registration 
+    - Consent Continuation Out 
+    - Conversion ( to change the business type of an organization eg: convert from a Member Funded Society to a BC Company)
```
Comment (chenhongjing) [inline docs/business.yaml:24]: Is it better to remove whitespace before "to change"?

File: docs/business.yaml [hunk 2]
```diff
@@ -3,8 +3,33 @@ info:
...
+    - Court Order Information
+    - Correction
+    - Dissolution
+    - Registration (sole proprietorship, general partnership)
```
Comment (chenhongjing) [inline docs/business.yaml:28]: And is it better to also capitalize first letters for SP/GP?

Comment (eason-pan-bc) [reply docs/business.yaml:28]: sounds good

**Review by AimeeGao — APPROVED** [review]
LGTM

File: docs/business.yaml [hunk 3]
```diff
@@ -3,8 +3,33 @@ info:
   title: businesses
   version: '2.0'
   description: |-
-    Businesses provideas all of the services needed to manage legal entities.
-    This encompasses all types of businesses authorized to do business by the Registrar.
+    The Business Registry is a centralized electronic register of all business in British Columbia, This encompasses all types of businesses authorized to do business by the Registrar. The Business API provides a secure means of integrating your business systems with the Registry.
```
Comment (AimeeGao) [inline docs/business.yaml:6]: The Business Registry is a centralized electronic register of all business in British Columbia, (Maybe it is better to change the ',' to '.' here)This encompasses all types of businesses authorized to do business by the Registrar.

File: docs/business.yaml [hunk 4]
```diff
@@ -3527,7 +3552,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/party'
       operationId: get-businesses-identifier-parties
-      description: Get all associated parties with a business.
+      description: Get all parties (Incorporatore, Completing Party and Directors) associated with a business.
```
Comment (chenhongjing) [inline docs/business.yaml:3555]: Typo, `Incorporator`

**Review by chenhongjing — APPROVED** [review]

**Pull request merged**